hexrd/instrument/detector.py

```diff
@@ -1131,14 +1131,6 @@ def make_powder_rings(
 
         # in case you want to give it tth angles directly
         if isinstance(pd, PlaneData):
-            # Okay, we have a PlaneData object
-            try:
-                pd = pd.makeNew()  # make a copy to munge
-            except TypeError:
-                # !!! have some other object here,
-                #     likely a dummy plane data object of sorts
-                raise
-
```

Comment:
I think this was being done before because they wanted a temporary version of the PlaneData object to modify (without modifying the PlaneData object that was passed in). See below where it is being modified. Are you sure this is safe to delete? It may have side effects.

Reply:
Not sure, I think Zack deleted that one.  It probably should be there

Reply:
I put it back.  I removed the useless try catch though.